google_guest_agent_packaging_diff_tool.py

```diff
@@ -1,46 +1,111 @@
-from pprint import pprint
 import os
 import click
+import pprint
+import requests
+import warnings
+
+from dataclasses import dataclass
 from debian import deb822
+from debian.deb822 import Deb822
+
+UBUNTU_CONTROL_FILENAME = "ubuntu_control.file"
+GOOGLE_CONTROL_FILENAME = "gce_control.file"
+
+
+@dataclass
+class ArchiveURLS:
+    ubuntu_control_url: str
+    google_upstream_control_url: str
+
+archives: dict[str, ArchiveURLS]
+
+archive_dict = {
+    "google-guest-agent": [
```

Comment:
You _could_ put these in alpha order to make them easier to find/maintain.

Reply:
Done, it's in the next upload :smile: